### PR TITLE
2016 01 25 Merge Orion's, Zsolt's and my changes together to bump the overall API version.

### PR DESCRIPTION
**Pull request by spencerpbartz**
When I walk the steps of .RST files I say to myself "only the penitent man will pass.... only the penitent man will pass"

### CODE REVIEW

**Comment by spencerpbartz** [comment]
It's a bit cryptic but any of your birds is welcome to review it!
@codingiam @jebentier @zmathe-sv @Inanepenguin @aschreifels.

Much appreciated!

File: source/api_documentation/call_outcomes_api/index.rst [hunk 1]
```diff
@@ -121,6 +121,7 @@ Endpoint:
   :description: Update a call outcome
   :page: update_call_outcome
```
Comment (spencerpbartz) [inline source/api_documentation/call_outcomes_api/index.rst]: Don't even ask why I had to put that space there.

File: source/api_documentation/call_outcomes_api/index.rst [hunk 2]
```diff
@@ -121,6 +121,7 @@ Endpoint:
...
 
+
```
Comment (spencerpbartz) [inline source/api_documentation/call_outcomes_api/index.rst]: Don't even ask why I had to put that space there.

File: source/api_documentation/transactions.rst [hunk 3]
```diff
@@ -1,792 +0,0 @@
-Transactions
```
Comment (spencerpbartz) [inline source/api_documentation/transactions.rst]: Kill the old page

File: README.md [hunk 4]
```diff
@@ -1,12 +1,17 @@
...
-2. You should no longer update the documentation that is present in the API controllers. This documentation will be removed shortly.
-3. A branch on git maps to a "version" on read-the-docs. To create a new version (branch) and have it displayed as default, you must go into read the docs admin and point "default" to the new branch. At that point, the old default will become an older version, so **do not** delete old branches of the form YYYY-MM-DD.
+2. The method of adding comments to API controllers is officially deprecated.
+3. A branch on git maps to a "version" on read-the-docs. To create a new version (branch) and have it displayed as default,
+ you must go into read the docs admin and point "default" to the new branch. At that point, 
+ the old default will become a legacy version that is still supported, so **do not** delete any old branches of the form YYYY-MM-DD.
 4. There is no master branch for this repo. If you are making a new version, you are responsible for setting the default branch of this repo to point to your new branch (version).
+5. Beware pitfalls. There are many. Dig into the shinx source code if need be.
```
Comment (Inanepenguin) [inline README.md]: shinx => sphinx ?

File: README.md [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
-3. A branch on git maps to a "version" on read-the-docs. To create a new version (branch) and have it displayed as default, you must go into read the docs admin and point "default" to the new branch. At that point, the old default will become an older version, so **do not** delete old branches of the form YYYY-MM-DD.
+2. The method of adding comments to API controllers is officially deprecated.
+3. A branch on git maps to a "version" on read-the-docs. To create a new version (branch) and have it displayed as default,
+ you must go into read the docs admin and point "default" to the new branch. At that point, 
+ the old default will become a legacy version that is still supported, so **do not** delete any old branches of the form YYYY-MM-DD.
 4. There is no master branch for this repo. If you are making a new version, you are responsible for setting the default branch of this repo to point to your new branch (version).
+5. Beware pitfalls. There are many. Dig into the shinx source code if need be. 
+6. Any Major changes involving a change to the toctree hierarchy  must be reviewed by MikeW, Spencer, or NickB.
```
Comment (Inanepenguin) [inline README.md]: toctree => doctree ?

File: README.md [hunk 6]
```diff
@@ -154,7 +159,12 @@ def source_handler(app, docname, source):
 ```
 
 #### Directive
+One of the harder parts of getting read the docs to work. Be ware of white space
```
Comment (Inanepenguin) [inline README.md]: Be ware => Beware ?

File: README.md [hunk 7]
```diff
@@ -154,7 +159,12 @@ def source_handler(app, docname, source):
 ```
 
 #### Directive
+One of the harder parts of getting read the docs to work. Be ware of white space 
+and trailing new lines. A home-made directive like shown below *must* have 
+two new lines after it (which comments counting as well).
```
Comment (Inanepenguin) [inline README.md]: I don't understand what `which comments counting as well` means. Do you mean "with" instead of "which"?

File: README.md (hunk 5 above)
Comment (spencerpbartz) [inline README.md]: toctree is what they call that nav menu generated thing on the left side. It's a directive in Sphinx.

File: README.md (hunk 7 above)
Comment (spencerpbartz) [inline README.md]: @Inanepenguin  Good catch!

File: README.md (hunk 6 above)
Comment (spencerpbartz) [inline README.md]: See what I did there? Easter egg!
Actually it was a typo. Good find.

File: source/api_documentation/transactions.rst [hunk 8]
```diff
@@ -1,792 +0,0 @@
...
-
-  * - transfer_from_type
-    - Transfer Type
-    - Where the call came from.
-
-  * - notes
-    - Notes
-    - Free-form notations on transaction.
-
-  * - call_outcome_outcome_name
```
Comment (nburwell) [inline source/api_documentation/transactions.rst]: (just so you have a placeholder item in the PR) as discussed these call_outcome_* fields are now missing

File: source/conf.py [hunk 9]
```diff
@@ -99,45 +103,70 @@ def build_template(match, template_file_name):
...
 def source_handler(app, docname, source):
+  # Build templates in custom_templates/ 
   source[0] = build_api_endpoint_template(source[0])
+  source[0] = build_tx_api_templates(source[0])
 
+  # Replace @@API_VERSION in templates with strings from doc_versions.py
   for symbol_string, version_string in VERSIONS.iteritems():
     source[0] = re.sub(symbol_string, version_string, source[0])
 
+  # TODO: If needed Add code here to Replace @@ORG_TYPE in templates with strings from org_types.py
```
Comment (mikeweaver) [inline source/conf.py]: This need the TODO?

File: source/org_types.py [hunk 10]
```diff
@@ -0,0 +1,18 @@
+ORG_TYPE_DOCNAMES = [
+  'network_user',
+  'advertiser_user',
+  'affiliate_user'
+]
+
+ORG_TYPES = {
+  '@@ORG_TYPE_PLURAL': [
+    'networks',
+	'advertisers',
```
Comment (mikeweaver) [inline source/org_types.py]: Indentation whitespace inconsistent.

**Comment by mikeweaver** [comment]
I commented on two little issues, otherwise good to do.

**Pull request merged**